andela/codango
# #131157821 Bug - logo on community creation

# What does this PR do?

It fixes a bug where images were not being uploaded to Cloudinary when creating a community
# Description of Task to be completed?
1. Make sure that logo is saved. Add tests for right dimension and the right size. Add tests for saving logo( Cloudinary mocking.)
2. Enforce size and dimension: When adding a logo during community creation please check that put a check that enforces a size check less than 36KB.  And also the 100 X 100 dimension.
# How should this be manually tested?
- Create an account and login with your username and password
- Click on the circled '+' button next to the COMMUNITIES  header
- Add the details as required on the **Create A New Community** form and choose an image. Submit the form.
- Click on 'View All' to see the community you've created
# What are the relevant pivotal tracker stories?
# 131157821 - Fix bugs with logo on community creation
# Screenshots

<img width="1277" alt="screen shot 2016-10-25 at 4 10 55 pm" src="https://cloud.githubusercontent.com/assets/20517534/19687366/ccbb1bc0-9acd-11e6-9048-b6f3b7c9f328.png">
